apps/src/templates/instructions/InlineAudio.jsx

```diff
@@ -175,9 +176,18 @@ const InlineAudio = React.createClass({
         <div
           className="inline-audio"
           style={this.props.style && this.props.style.wrapper}
+          onMouseOver={function (){
+            $("#volume").css("background-color", color.cyan);
+            $(".playPause").css("background-color", color.cyan);
+          }}
+          onMouseOut={function (){
+            $("#volume").css("background-color", color.lightest_purple);
+            $(".playPause").css("background-color", color.lightest_purple);
+          }}
```

Comment:
you can actually do this with Radium! check out https://github.com/FormidableLabs/radium#usage for an example